OpenRA.Mods.Cnc/Effects/SatelliteLaunch.cs

```diff
@@ -45,6 +46,8 @@ public void Tick(World world)
 				var palette = info.SatellitePaletteIsPlayerPalette ? info.SatellitePalette + launcher.Owner.InternalName : info.SatellitePalette;
 				world.AddFrameEndTask(w => w.Add(new GpsSatellite(world, pos, info.SatelliteImage, info.SatelliteSequence, palette, info.RevealDelay * 25, launcher.Owner)));
 			}
+
+			world.ScreenMap.Update(this, pos, doors.Image);
```

Comment:
Nit: can you put this immediately after `doors.Tick()` so that it's clear they two are related?